parser/src/cfg/mod.rs

```diff
@@ -236,7 +236,7 @@ pub const BOOLEAN_VALUES: [&str; 6] = ["yes", "true", "1", "no", "false", "0"];
 const DEF_LOCAL_KEYS: &str = "deflocalkeys-win";
 #[cfg(all(feature = "interception_driver", target_os = "windows"))]
 const DEF_LOCAL_KEYS: &str = "deflocalkeys-wintercept";
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "linux", target_os = "unknown"))]
```

Comment:
If I understand correctly `deflocalkeys-*` can't be simply applied *after* the config is parsed because https://github.com/jtroo/kanata/blob/e7988d8f842a64b5ada009dff9ba0cd0820d6405/parser/src/cfg/mod.rs#L759 needs to be called *before* using `str_to_oscode`. So as a workaround for now I'll just do this simple change instead.